napari/layers/labels/_tests/test_labels_mouse_bindings.py

```diff
@@ -0,0 +1,101 @@
+import numpy as np
+import collections
+from napari.layers import Labels
+from napari.utils.interactions import (
+    ReadOnlyWrapper,
+    mouse_press_callbacks,
+    mouse_move_callbacks,
+    mouse_release_callbacks,
+)
+
+
+def test_paint():
+    """Test painting labels with different brush sizes."""
+    data = np.ones((20, 20))
+    layer = Labels(data)
+    layer.brush_size = 10
+    layer.mode = 'paint'
+    layer.selected_label = 3
+    layer.position = (0, 0)
+
+    Event = collections.namedtuple('Event', 'type is_dragging')
```

Comment:
I really don't understand what you're doing with this... Could you add a comment about how this is mocking a real event? `'type is_dragging'` is particularly confusing to me...

Reply:
yeah, that's the vispy event syntax. @kne42 proposal in #1048 will remove this, but it will be much easier to make those changes after this refactor than before. I'll add a comment.